Classes/Controller/MapController.php

```diff
@@ -181,6 +181,10 @@ public function showAction(Map $map = null): ResponseInterface
         // get current map
         /* @var Map $map */
         $map = $map ?? $this->mapRepository->findByUid($this->settings['map']);
+        
+        if (is_null($map)) {
+            return;
```

Comment:
The return statement does not match the function return type ( ResponseInterface ).